server/views/feedbacks/add.pug

```diff
@@ -0,0 +1,21 @@
+extends ../layout/index.pug
+block content
+  if addResult.success
+    .alert.alert-success
+      | Sucessfully added!
+  else if addResult.error
+    .alert.alert-danger
+      | Something is wrong! Please fill all fields.
+  form(action='/feedbacks', method='POST')
+    .form-row
+    .form-group
+      label(for='inputName') Name
+      input#inputName.form-control(type='text', placeholder='Name', name='uaName')
```

Comment:
Gread job Nadia. Could you please change 'uaName' to 'faName'. 'faName' stands for 'feedback add - Name' field.

Reply:
And change all dependent pieces.